### DISCUSSION TITLE
Downloading a CSV file and the data keeps replicating forever

### DISCUSSION THREAD

**Discussion by cplabsafety** [iterate-ch/cyberduck#14861] [Q&A] [answered]
I've created dozens of csv product exports from my BigCommerce store directly into Cyberduck. About 2 years ago,  I began seeing a glitch that has continued to this day on all my exports. Instead of downloading the data one time, it downs all the rows repeatedly, until it reaches about a million rows, and then fails.
When I look at the file in Cyberduck it shows a file around 226 MB which makes sense for 100K-300K rows of product. Once I click "download" the file downloads the data repeatedly on the same sheet until reaches a file size of 10-12 GB, and then times out. Sometimes I can open it in Excel and remove duplicates, which removes 800,000+ rows and appears the way it was intended. Other times the file is too large for Excel to open, and I cannot access my data at all. It's becoming quite a problem and I haven't found anyone talking about this issue online, so I don't know whether it's common or unique. I've reinstalled Cyberduck many times without a fix. This happens on at least 2 different computers as well, both on Windows 10 and 11.  Is there a setting to force downloads to abort once they've downloaded the first set of data? Something is triggering a repeat, and BigCommerce tech support denies any knowledge of it, claiming it's a Cyberduck issue.
Note: other files download correctly, both from BigCommerce and other sources. Only large product exports (spreadsheets with 100K + rows) have this glitch.

**Comment by dkocher** [comment, marked as answer]
Please try if unchecking _Preferences → Transfers → Segmented downloads with multiple connections per file_ resolves the problem.

**Reply by cplabsafety** [reply]
Thank you, this worked! What a relief.
